cocos/asset/assets/image-asset.ts

```diff
@@ -163,7 +163,7 @@ function fetchImageSource (imageSource: ImageSource) {
 
 // 返回该图像源是否是平台提供的图像对象。
 function isNativeImage (imageSource: ImageSource): imageSource is (HTMLImageElement | HTMLCanvasElement | ImageBitmap) {
-    if (ALIPAY || TAOBAO || TAOBAO_MINIGAME || XIAOMI || BAIDU || WECHAT) { // TODO(xwx):WECHAT only for wechat miniprogram, need to refactor
+    if (ALIPAY || TAOBAO || TAOBAO_MINIGAME || XIAOMI || BAIDU || WECHAT_MINI_PROGRAM) {
```

Comment:
The same as above.

Reply:
This logic only for wechat mini program(WECHAT_MINI_PROGRAM). not for wechat minigame(WECHAT)